rpc/scanner/service.proto

```diff
@@ -30,4 +30,5 @@ message ScanResponse {
 message Result {
   string   target                               = 1;
   repeated common.Vulnerability vulnerabilities = 2;
-}
+  string                        type            = 3;
```

Comment:
Did you apply `make fmt`? It's a bit weird compared to `target`.

Reply:
Yes I ran `make fmt`

Reply:
OK.